info.json

```diff
@@ -177,11 +177,11 @@
 			"name": "Miquel ViR",
 			"bio": [],
 			"volunteer": {
+				"code": {},
```

Comment:
@miquelvir Code volunteering is done after demonstrated work on Flask. On your Github we could not detect great Flask contributions. I suggest putting it off until you have public contributions. The Traveller soft i mentionned is a help area that we need until we accomodate more plugins. Else you can also add code when you become a consistent contributor to a popular and useful extention somewhere. This is done to ensure that people listed on the page are actually backed by publicly demonstrable pieces of work.

Reply:
Sure. I thought that since you added me as maintainer it made sense. No problem!